CWE/events/East Turkestan Additions.txt

```diff
@@ -0,0 +1,1336 @@
+		any_pop = {
+			limit = { culture = uighur }
+			ideology = { factor = 0.10 value = communist }
+			militancy = -8
+			consciousness = -8
+		}
+		XIJ = { country_event = 8013812 }
+		ai_chance = { 
+			factor = 0.5
+			modifier = {
+				factor = 10
+				has_country_flag = ets_tore
+			}
+		}
+	}
+}
+
+#Accept - peace
+country_event = {
+	id = 8013811
+	title = "East Turkestan Accepts"
+	desc = EVTDESC8013811
+	picture = "ets_kmt2"
+
+	is_triggered_only = yes
+
+	option = {
+		name = "We can focus on ensuring a lasting peace."
+		end_war = ETS
+		prestige = 5
+		any_pop = {
+			militancy = -3
+			consciousness = -3
+		}
+	}
+}
+
+#Reject - Continued war
+country_event = {
+	id = 8013812
+	title = "East Turkestan Refuses"
+	desc = EVTDESC8013812
+	picture = "ets_army"
+
+	is_triggered_only = yes
+
+	option = {
+		name = "The prospect of a peaceful Xinjiang dies with their refusal."
+		prestige = -10
+		any_pop = {
+			militancy = 3
+			consciousness = 3
+		}
+	}
+}
+
+#Coalition Government proposed
+country_event = {
+	id = 8013813
+	title = "Proposal for a Coalition Government"
+	desc = EVTDESC8013813
+	picture = "turkestan_coalition"
+
+	is_triggered_only = yes
+
+
+	option = {
+		name = "We will move forward together! (Game Over)"
+		badboy = -3
+		XIJ = { country_event = 8013814 }
+		ai_chance = { 
+			factor = 0.8
+		}
+	}
+
+	option = {
+		name = "We cannot accept this compromise!"
+		badboy = 3
+		add_country_modifier = { name = political_strife duration = 1095 }
+		XIJ = { country_event = 8013815 }
+		ai_chance = { 
+			factor = 0.2
+		}
+	}
+}
+
+#Accept - Game over
+country_event = {
+	id = 8013814
+	title = "East Turkestan Accepts Coalition Offer"
+	desc = EVTDESC8013814
+	picture = "turkestan_coalition"
+
+	is_triggered_only = yes
+
+
+	option = {
+		name = "Xinjiang is united once more!"
+		prestige = 10
+		inherit = ETS
+		change_capital = 1600
+		add_accepted_culture = uighur
+		1600 = {
+			change_province_name = "Urumqi"
+		}
+		any_pop = {
+			militancy = -3
+			consciousness = -3
+		}
+	}
+}
+
+#Reject - war
+country_event = {
+	id = 8013815
+	title = "East Turkestan Rejects Coalition Offer"
+	desc = EVTDESC8013815
+	picture = "ets_army"
+
+	is_triggered_only = yes
+
+
+	option = {
+		name = "Then we must ensure peace through force."
+		prestige = -10
+		any_pop = {
+			militancy = 3
+			consciousness = 3
+		}
+		war = {
+			target = ETS
+			attacker_goal = { casus_belli = civil_war }
+			defender_goal = { casus_belli = civil_war }
+			call_ally = no
+		}
+	}
+}
+
+#Proclaimation of an East Turkestani People's Republic
+country_event = {
+   id = 8013816
+   title = "East Turkestan Triumphant"
+   desc = EVTDESC8013816
+   picture = "ets_communist"
+
+	fire_only_once = yes
+
+	trigger = {
+		tag = ETS
+		XIJ = { exists = no }
+		NOT = { has_country_flag = ets_victory }
+	}
+
+	mean_time_to_happen = { months = 1 }
+
+	option = {
+		name = "Proclaim a East Turkestani People's Republic!"
+		government = proletarian_dictatorship

```

Comment:
Please ensure that no government changes are done through event unless the country was independent

Correct way to change gov:
government = proletarian_dictatorship
country_event = 800052
ruling_party_ideology = communist